sdk/automanage/Azure.ResourceManager.Automanage/src/autorest.md

```diff
@@ -0,0 +1,177 @@
+# Generated code configuration
+Run `dotnet build /t:GenerateCode` to generate code.
+``` yaml
+save-inputs: true
+azure-arm: true
+csharp: true
+library-name: automanage
+namespace: Azure.ResourceManager.automanage
+# require: https://github.com/Azure/azure-rest-api-specs/blob/d32cece9ca8814ef42085d4bbc426dc35bbcaf87/specification/automanage/resource-manager/readme.md
+require: https://github.com/AndrewCS149/azure-rest-api-specs/blob/360b7c3331b919bfe90148adff6abe025d21c720/specification/automanage/resource-manager/readme.md
```

Comment:
The sdk should use a file from [github.com/Azure/azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs), not personal repo

Reply:
Oops! I meant to switch it. Done!